### ISSUE TITLE
Figure out changes required to upgrade to iNZightPlots 2.15

### ISSUE THREAD

**Issue by tmelliott** [iNZightVIT/Lite#287]
https://github.com/iNZightVIT/iNZightPlots/releases/tag/2.15.0

Issues (so far) seem to be the way data is passed to the `inzplot()` function, resulting in errors.

**Comment by tmelliott** [comment]
This is fixed in #391?

**Comment by chrk623** [comment]
> This is fixed in [#391](https://github.com/iNZightVIT/Lite/pull/391)?

Yes. New TS module was also updated to use `iNZightTS:::plot.inz_ts` (#372), not sure what it uses under the hood though. 
I see a mention of multiple response, `inzplot` supports it?
